src/modules/todos.js

```diff
@@ -0,0 +1,100 @@
+import CompletedTasksStatus from './tasksCompleted.js';
+
+const status = new CompletedTasksStatus();
+export default class Tasks {
+  constructor() {
+    this.tasksArray = JSON.parse(localStorage.getItem('tasks')) || [];
+  }
+
+  saveAndRender = () => {
+    localStorage.setItem('tasks', JSON.stringify(this.tasksArray));
+    this.displayList();
+  }
+
+  displayList = () => {
+    // display tasks
+    const todoContainer = document.querySelector('#todo-list');
+    todoContainer.innerHTML = '';
+    this.tasksArray.forEach((task) => {
+      const li = document.createElement('li');
+      li.className = 'todo-task';
+      li.id = task.index;
+      li.innerHTML = `<div><button class="check-task"><i class="fa-regular fa-square"></i><i class="fa-solid fa-check"></i></button> <input class="todo-input" type="text" value="${task.description}"></div><button><i class="fa-solid fa-ellipsis-vertical" style="margin-left:15rem;"></i>
```

Comment:
-The `three-dot button` does not function properly on project output.

Reply:
Hi @gergitokusse ,
Thanks for reviewing the code and positive feedback, yes there is no any edit and delete options and they are not visible, but when we focus on the three-dost they will properly work 😃 
<img width="299" alt="error" src="https://github.com/Zohra-Neda/To-Do-List-Review/assets/58807251/32163425-74b2-4c9d-9099-7c56ab570471">
Additionally, when the user click on the delete icon it will re-arrange all array indexes 💯 
<img width="295" alt="error1" src="https://github.com/Zohra-Neda/To-Do-List-Review/assets/58807251/dbc1baf7-953a-4445-b8fc-993a68b40b40">

Reply:
## I got it 🚚🚚